.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @bai
```

Comment:
wondering if this can be the team of sarama maintainers? (if there is such a team ?)

Reply:
There's no such team, but I'll be happy to amend this list — arguably having a list of usernames here is easier than creating a proper GitHub team.